### PR TITLE
Various fixes/updates

### LINKED ISSUE #25: Remove all passwords before user deletion
https://github.com/keycloud/keycloud/blob/1719cc8849f68799b50da29e85c2fc6ea1b199bc/server/crudHandler.go#L109

for referencing

### LINKED ISSUE #26: Username not updating
https://github.com/keycloud/keycloud/blob/1719cc8849f68799b50da29e85c2fc6ea1b199bc/server/crudHandler.go#L118
I receive the successful response but in database the username is not updated.

### LINKED ISSUE #32: Add mail to return object of get user request
Add mail string to the response of get user request  
specifically: in crudHandler.go at line 148 - 154

### PR DESCRIPTION
**Pull request by Dudeldu**
Fixes #25
Fixes #26 
Fixes #32

### CODE REVIEW

File: server/docker-compose.yml [hunk 1]
```diff
@@ -13,6 +13,8 @@ services:
       - $POSTGRES_PORT:$POSTGRES_PORT
     networks:
       - app-network
+    volumes:
+      - ${PWD}/init_db.sql:/docker-entrypoint-initdb.d/init_db.sql
```
Comment (Dudeldu) [inline server/docker-compose.yml:17]: Maybe use absolute path

File: README.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://teamcity.zeekay.dev/app/rest/builds/buildType(id:Keycloud_Build)/statusIcon)](https://teamcity.zeekay.dev/app/rest/builds/buildType(id:Keycloud_Build)/statusIcon)
```
Comment (hottek) [inline README.md:1]: Nice!

File: server/docs/backend-calls.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # Backend calls
 | Method | Route | Description | Body | Requires Cookie | Return | Additional Information
 |---|---|---|---|---|---|---|
-| GET | `/user` | retrieves username and password | - | ✔️ | `{"username": "johndoe", "masterpassword": "my-master-passwd"}` |
-| DELETE | `/user` | deletes user | - | - | ✔️ | `{"Status": "REMOVED", "Error": ""}`| [remove passwords before user](https://github.com/keycloud/keycloud/issues/25) |
-| PUT | `/user` |  updates username | `{"username": "newjohndoe"}` | ✔️ | - | [user updating not working](https://github.com/keycloud/keycloud/issues/26) |
```
Comment (hottek) [inline server/docs/backend-calls.md:6]: would be nice if the user could update his mail

File: server/docker-compose.yml (hunk 1 above)
Comment (zkdev) [reply server/docker-compose.yml:17]: tried already, will give it another run ...

Comment (Dudeldu) [reply server/docker-compose.yml:17]: This requires the docker compose file to be run inside the server directory

**Pull request merged**